windowsforms/FlexGridShowcaseDemo/cs/MainForm.cs

```diff
@@ -0,0 +1,682 @@
+﻿using C1.Win.FlexGrid;
+using C1.Win.Themes;
+using C1.Win.Ribbon;
+using C1.Win.RulesManager;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Data;
+using System.Drawing;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Windows.Forms;
+
+namespace FlexGridShowcaseDemo
+{
+
+    public partial class MainForm : C1RibbonForm 
+    {
+        private const string _customThemeName = "Greenwich";
+        private const int _footerTextPadding = 4;
+        private DataSet _dataSet = new DataSet();
+        private C1RulesManager _rulesManager = new C1RulesManager();
+        private IEnumerable<IRule> _rules = Enumerable.Empty<IRule>();
+        private static Random _rnd = new Random();
```

Comment:
The convention is to use `s_` for static fields. Also, because this is .NET 5.0 app, we can use C# 9 target-typed new. Not sure if we will require a temporary LangVersion set to Preview until C# 9 is released or not.

```suggestion
        private static Random s_rnd = new();
```

Reply:
Fixed with 9df46856528d72d8521322c13e00a0ff0264672c.